docs/settings/logging-settings.asciidoc

```diff
@@ -4,6 +4,18 @@
 <titleabbrev>Logging settings</titleabbrev>
 ++++
 
+{kib} relies on three high-level entities to set the logging service: _appenders_, _loggers_ and _root_.
+
+__Appenders__ define where log messages are displayed (e.g. stdout or console) and stored (eg. file on the disk), and their layout.
```

Comment:
Eliminate the e.g., which is not good for internationalization. For example:

Appenders define where log messages are displayed (stdout or console), in which file they are stored, and their layout.

Reply:
Done! thanks for the tip!